modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+        _mouseOnPair = nullptr;
+    }
+    else {
+        _mouseOnPair = it->get();
+    }
+
+    // Selection has changed
+    if (previousPair != _mouseOnPair) {
+        // Remove highlight
+        if (previousPair) {
+            previousPair->removeHighlight(_highlightAddition);
+        }
+        // Add highlight to new selection
+        if (_mouseOnPair) {
+            _mouseOnPair->highlight(_highlightAddition);
+        }
+    }
+}
+
+void SkyBrowserModule::addTargetBrowserPair(const std::string& targetId, const std::string& browserId) {
+    ScreenSpaceSkyTarget* target = dynamic_cast<ScreenSpaceSkyTarget*>(
+        global::renderEngine->screenSpaceRenderable(targetId)
+    );
+    ScreenSpaceSkyBrowser* browser = dynamic_cast<ScreenSpaceSkyBrowser*>(
+        global::renderEngine->screenSpaceRenderable(browserId)
+    );
+    
+    // Ensure pair has both target and browser
+    if (browser && target) {
+        _targetsBrowsers.push_back(std::make_unique<TargetBrowserPair>(browser, target));
+    }
+}
+
+void SkyBrowserModule::removeTargetBrowserPair(const std::string& id) {
+    TargetBrowserPair* found = getPair(id);
+    if (!found) {
+        return;
+    }
+
+    auto it = std::remove_if(
+        _targetsBrowsers.begin(), 
+        _targetsBrowsers.end(),
+        [&](const std::unique_ptr<TargetBrowserPair>& pair) {
+            return *found == *(pair.get());
+        }
+    );
+
+    _targetsBrowsers.erase(it, _targetsBrowsers.end());
+    _mouseOnPair = nullptr;
+}
+
+void SkyBrowserModule::lookAtTarget(const std::string& id) {
+    TargetBrowserPair* pair = getPair(id);
+    if (pair) {
+        startRotatingCamera(pair->targetDirectionGalactic());
+    }
+}
+    
+void SkyBrowserModule::setHoverCircle(ScreenSpaceImageLocal* circle) {
+    _hoverCircle = circle;
+}
+
+void SkyBrowserModule::moveHoverCircle(int i) {
+    const ImageData& image = _dataHandler->getImage(i);
+
+    // Only move and show circle if the image has coordinates
+    if (_hoverCircle && image.hasCelestialCoords && _isCameraInSolarSystem) {
+        // Make circle visible
+        _hoverCircle->setEnabled(true);
+
+        // Set the exact target position 
+        glm::dvec3 localCamera = skybrowser::equatorialToLocalCamera(image.equatorialCartesian);
+
+        if (_hoverCircle->isUsingRaeCoords()) {
+            glm::vec3 position = _hoverCircle->raePosition().x * glm::vec3(localCamera);
+            _hoverCircle->setRaeFromCartesianPosition(position);
+        }
+        else {
+            _hoverCircle->setCartesianPosition(
+                skybrowser::localCameraToScreenSpace3d(localCamera)
+            );
+        }
+    }
+}
+
+void SkyBrowserModule::disableHoverCircle() {
+    if (_hoverCircle && _hoverCircle->isEnabled()) {
+        _hoverCircle->setEnabled(false);
+    }
+}
+
+void SkyBrowserModule::loadImages(const std::string& root, 
+                                  const std::filesystem::path& directory) {
```

Comment:
since this funciton signature is two line, the starting bracket should be moved to it's own line: 
```
void SkyBrowserModule::loadImages(const std::string& root, 
                                  const std::filesystem::path& directory) 
{
   ...
```